### PR TITLE
Feat/redirect filter active

### PR DESCRIPTION
**Pull request by jleveugle**

### CODE REVIEW

File: packages/manager/apps/public-cloud/src/index.routes.js [hunk 1]
```diff
@@ -1,3 +1,45 @@
-export default /* @ngInject */ ($urlRouterProvider) => {
-  $urlRouterProvider.otherwise('/pci/projects');
+const DEFAULT_PROJECT_KEY = 'PUBLIC_CLOUD_DEFAULT_PROJECT';
+
+export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
+  $stateProvider
+    .state('app', {
+      url: '/',
+      resolve: {
+        redirectTo: /* @ngInject */ ($q, ovhUserPref, publicCloud) => publicCloud
```
Comment (marie-j) [inline packages/manager/apps/public-cloud/src/index.routes.js]: If alphabetical order is possible :)

File: packages/manager/apps/public-cloud/src/index.routes.js [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+                    // No project is defined as favorite
+                    // Go on the first one :)
+                    return {
+                      state: 'pci.projects.project',
+                      stateParams: {
+                        projectId: projects[0].project_id,
+                      },
+                    };
+                  }
+                  // [TODO] Go to error page
```
Comment (marie-j) [inline packages/manager/apps/public-cloud/src/index.routes.js]: 😄

File: packages/manager/modules/pci/src/projects/projects.routing.js [hunk 3]
```diff
@@ -4,14 +4,15 @@ import template from './projects.html';
...
       url: '/projects',
       controller,
       controllerAs: '$ctrl',
       template,
-      resolve: {
-        breadcrumb: /* @ngInject */ $translate => $translate
-          .refresh()
-          .then(() => $translate.instant('pci_projects')),
-      },
+      // resolve: {
```
Comment (cbourgois) [inline packages/manager/modules/pci/src/projects/projects.routing.js]: should be removed

File: packages/manager/apps/public-cloud/src/components/redirectTo/index.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import angular from 'angular';
```
Comment (cbourgois) [inline packages/manager/apps/public-cloud/src/components/redirectTo/index.js]: Could be a external isolated module ? 🤔

File: packages/manager/apps/public-cloud/src/index.service.js [hunk 5]
```diff
@@ -6,11 +6,17 @@ export default class PublicCloud {
     this.iceberg = iceberg;
   }
 
-  getProjects() {
-    return this
+  getProjects(filters = []) {
+    let promise = this
```
Comment (marie-j) [inline packages/manager/apps/public-cloud/src/index.service.js]:
```suggestion
    const promise = filters.reduce((promise, filter) => 
    promise.addFilter(filter),
    this.iceberg('/cloud/project'))
```
?

**Pull request merged**